### PR TITLE
[C-2790] Add private DogEar to desktop playlist cards

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds `privote` `DogEar` to all playlist cards
Refactors all usages of `BannerIcon` to `DogEar`
Updates card and upload-chip to use `Tile`
Updates `Tile` to only have hover effect when `onClick` or `href` present
Adds `verify` command at root, which quickly runs typecheck and lint in web/mobile. Ive found myself running this so many times at this point.

### CODE REVIEW

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2790-add-hidden-playlist-indicator

File: packages/mobile/package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
     "typecheck:trace": "tsc --generateTrace tsc_trace",
     "clean": "react-native-clean-project",
     "preinstall": "npm-force-resolutions",
-    "postinstall": "patch-package"
+    "postinstall": "patch-package",
+    "verify": "concurrently \"npm:typecheck\" \"npm:lint:fix\""
```
Comment (amendelsohn) [inline packages/mobile/package.json:43]: ooh thank you!

File: package.json [hunk 2]
```diff
@@ -31,7 +31,8 @@
     "stems": "cd packages/stems && npm run start",
     "SDK========================================": "",
     "sdk:update": "lerna exec -- npm-check-updates @audius/sdk -u --target @beta && rm -rf packages/*/node_modules/@audius/sdk && npm i",
-    "sdk:link": "./scripts/link-sdk.sh"
+    "sdk:link": "./scripts/link-sdk.sh",
+    "verify": "lerna run --scope=audius-client --scope audius-mobile-client verify"
```
Comment (amendelsohn) [inline package.json:35]: does this run on both web and mobile?
curious why the scopes are using different syntax `--scope=x vs --scope x`

File: packages/web/src/components/card/desktop/Card.module.css [hunk 3]
```diff
@@ -1,28 +1,9 @@
...
-    0 2px 5px -2px var(--tile-shadow-3);
-  transition: all 0.18s ease-in-out;
-}
-
-.cardContainer:hover {
-  background-color: var(--neutral-light-10);
-  box-shadow: 0 1px 5px 1px var(--tile-shadow-1-alt),
-    0 1px 0 0 var(--tile-shadow-2), 0 2px 10px -2px var(--tile-shadow-3);
-  transform: scale3d(1.01, 1.01, 1.01);
-}
```
Comment (amendelsohn) [inline packages/web/src/components/card/desktop/Card.module.css:20]: Confirming that we don't need the hover states anymore?

File: packages/web/src/components/upload/UploadChip.module.css [hunk 4]
```diff
@@ -1,69 +1,42 @@
...
-  padding: 32px;
-  margin-top: 0px;
-}
-
-.uploadChip:hover {
-  background-color: var(--neutral-light-10);
-  box-shadow: 0 1px 5px 1px var(--tile-shadow-1-alt),
-    0 1px 0 0 var(--tile-shadow-2), 0 2px 10px -2px var(--tile-shadow-3);
-  transform: scale(1.01);
-}
```
Comment (amendelsohn) [inline packages/web/src/components/upload/UploadChip.module.css:30]: removing hover behavior ok?

**Review by amendelsohn — APPROVED** [review]
🐶 👂

File: packages/web/src/components/card/desktop/Card.module.css (hunk 3 above)
Comment (dylanjeffers) [reply packages/web/src/components/card/desktop/Card.module.css:20]: yeah they all come from the tile now

File: packages/web/src/components/upload/UploadChip.module.css (hunk 4 above)
Comment (dylanjeffers) [reply packages/web/src/components/upload/UploadChip.module.css:30]: same!

**Pull request merged**